pkg/ccl/backupccl/targets.go

```diff
@@ -209,6 +209,56 @@ func getAllDescChanges(
 	return res, nil
 }
 
+// blockMultiRegionTableBackup validates that for all tables included in the
+// backup, their parent database is also being backed up.
+func blockMultiRegionTableBackup(
```

Comment:
nit: i'd call this `checkNotMultiRegionTableBackup`, as `block` for me implies this will block everything